local-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+rm -f com.sweethome3d.Desktop.flatpak
+rm -rf _build ; mkdir _build
+rm -rf _repo ; mkdir _repo
```

Comment:
Is there a reason for this? once the app is on Flathub the bots are doing a great job so far.

Reply:
I have this for every one of the packages I maintain. It stops me from having to remember all the steps. And the bot don't run on my machine...